Dockerfile

```diff
@@ -7,16 +7,17 @@ FROM ${BASE_IMAGE_BUILDER}:${GO_VERSION}-alpine${ALPINE_VERSION} AS builder
 ARG GOARCH=amd64
 ARG GOARM
 ARG VERSION=0.2.4
```

Comment:
could we pass this in as an argument as well? Something like `--build-arg VERSION=$(git describe --abbrev=0 --tag)`

Reply:
well yes you can, and it will override the `ARG VERSION=0.2.4`, the one set here is just the default one if none is given. I can set it to nothing if you prefer?

Reply:
I reckon we should leave VERSION to be blank if not explicitly passed, so that we don't incorrectly think that a person was using version 0.2.4 when they raise an issue.

The automated build on docker hub sounds like a good idea, and I'm sure it's possible with circle CI. I'll make an issue for that

Reply:
https://github.com/jesseduffield/lazydocker/issues/66

Reply:
It is worth noting that things like `--build-arg VERSION=$(git describe --abbrev=0 --tag)` (or any other value that relies on certain `git` commands to extract a value) won't work when the build context is the Git URL and not built from a local clone or the repo. This is not something that is suggested in the modified README of this PR.

Reply:
Hi there! That seems like an odd bug from Docker, are you sure about it?

I'm waiting for @jesseduffield 's Docker Hub username and Docker Hub image name so that I update (push) a new readme in which
- the user just pulls and run the Docker image from Docker Hub, there is required building (except for ARM)
- the developer/contributor git clones and then docker builds locally, with its build arguments such as `VERSION`, it's cleaner than building from the `.git` URL.

Reply:
Actually that's a fair point, @jesseduffield  maybe having default values for build arguments (such as `VERSION=0.2.4`) would be better, if someone builds from the git repository?

Reply:
even so, any static values we give for those will be quickly outdated. Do you mean we should make it try to grab the current version when building it locally? I wonder how many people would opt to do that as opposed to just pulling the image from docker hub.

BTW the docker hub username is `lazyteam` and the repository is `lazydocker`